Riot/Managers/Analytics/DecryptionFailureTracker.m

```diff
@@ -24,6 +24,8 @@
 // and reporting them as failures
 #define GRACE_PERIOD 60
 
+// E2E failures analytics category.
+NSString *const DecryptionFailureTrackerAnalyticsCategory = @"e2e.failure";
```

Comment:
If this constant is only used internally you use the `k` prefix: `kDecryptionFailureTrackerAnalyticsCategory`.